ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -119,16 +119,19 @@ export class AccountCreationSurvey extends React.Component<AccountCreationSurvey
     const errors = validate(demographicSurvey, validationCheck);
 
     return <div style={{marginTop: '1rem', paddingLeft: '3rem', width: '26rem'}}>
-      <label style={{color: colors.primary, fontSize: 16}}>
-        Please complete Step 2 of 2
-      </label>
-      <ListPageHeader>
-        Demographics Survey <label style={{fontSize: '12px', fontWeight: 400}}>
-        (All Survey Fields are optional)</label>
-      </ListPageHeader>
-
+      <TextColumn>
+        <div style={{fontSize: 28, fontWeight: 400, marginBottom: '.8rem'}}>Optional Demographics Survey</div>
+        <div style={{fontSize: 16, marginBottom: '.5rem'}}>
+          Please complete Step 2 of 2
+        </div>
+        <div>
+          <label style={{fontWeight: 600}}>Answering these questions is optional.</label> The <i>All of Us </i>
```

Comment:
I think you could use the `<AoUTitle/>` object we have for this? (I think that one is 'Research Program' and not 'Researcher Workbench'

Reply:
👍